### ISSUE TITLE
Apple M1 Chip: R crashes when checking if 'transformers' is available

### ISSUE THREAD

**Issue by Sdbock** [AlexChristensen/transforEmotion#2]
Hello,

This is likely a reticulate issue, but every time I check to see if the transformers module is available, R crashes. The module has definitely been installed correctly. Any idea why this function (or reticulate, likely) would be crashing my R sessions?

Thanks!
-Sean

**Comment by Sdbock** [comment]
Update: Figured out the issue had to do with incompatibility because of the Apple M1 chip. I had to install new versions of R and Python compatible with M1 chip. After doing that, reticule works okay, but there's still an issue with installing the transforms module, due to yet another M1 incompatibility with Rust, which is needed for tokenizers. After separately installing a compatible version of Rust, the transforms module finally installed correctly, and everything runs fine now.
